### PR TITLE
Add phpstan wordpress extension

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,9 @@
     "phpunit/phpunit": "^7.5",
     "brain/monkey": "^2.3",
     "giacocorsiglia/wordpress-stubs": "^5.1",
```
Comment (chesio) [inline composer.json]: @szepeviktor I guess, this depencency should be removed as well?

Comment (szepeviktor) [reply composer.json]: Oh yesss!

Comment (szepeviktor) [reply composer.json]: Done.

**Comment by chesio** [comment]
Thanks @szepeviktor!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**